server/controllers/chartsController.js

```diff
@@ -33,6 +33,10 @@ const postProcessData = (data, studyTotals) => {
           ...existingEntriesForStudy.percentages,
           [valueLabel]: percent,
         },
+        targets: {
```

Comment:
what's the reason for this change? I thought we already had access to the study totals/targets?

Reply:
There's a studyTotals object and that has the summation of the sites targets, this change adds the individual target of the value so that when we display a row of a value we can plug in the individual target to match the Value/Target(%) piece. It is currently missing from the charts data object and the above is the reason why it was added.

Reply:
<img width="287" alt="Screen Shot 2022-10-03 at 3 17 54 PM" src="https://user-images.githubusercontent.com/19805355/193661857-88d68d29-9d27-4b2d-8fa7-0d71a5858c5d.png">